apps/constellation/manifest_builder.cpp

```diff
@@ -64,7 +64,7 @@ void GenerateDefaultManifest(std::string const &external_address, uint16_t port,
   // register all of the lanes (storage shards)
   for (uint32_t i = 0; i < num_lanes; ++i)
   {
-    peer.Update(external_address, static_cast<uint16_t>(port + STORAGE_PORT_OFFSET + i));
+    peer.Update(external_address, static_cast<uint16_t>(port + STORAGE_PORT_OFFSET + (2 * i)));
```

Comment:
I don't think I understand this change. Can you let me know f2f what's going on here?

Reply:
Just an inconsistency from an older piece of the logic. Each lane shard has 2 network inferfaces so this is represented here.